pkg/detector/library/testdata/fixtures/python.yaml

```diff
@@ -0,0 +1,8 @@
+- bucket: "python::GitHub Security Advisory Pip"
```

Comment:
Where is this file used?

Reply:
It's part of a missing test case that was added here https://github.com/aquasecurity/trivy/pull/709/commits/336917b5f2040bbb724c0c75b87c69a0a6864515#diff-fb318d27d837f7ff34aaab41eb90af4ef9f7359bd6dc8a0d23e5718ea5ef608aR50-R66